src/App.css

```diff
@@ -1,41 +1,39 @@
-#root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
-  text-align: center;
+* {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-weight: inherit;
+	font-style: inherit;
+	font-size: 100%;
+	font-family: inherit;
+	vertical-align: baseline;
+	box-sizing: border-box;
 }
 
-.logo {
-  height: 6em;
-  padding: 1.5em;
-  will-change: filter;
-}
-.logo:hover {
-  filter: drop-shadow(0 0 2em #646cffaa);
-}
-.logo.react:hover {
-  filter: drop-shadow(0 0 2em #61dafbaa);
+:root {
+	--color-primary: #99c1b9;
+	--color-secondary: #8e7dbe;
+	--color-tertiary: #f2d0a9;
+  --color-error: #d88c9a;
+	--color-font-primary: #000000;
+	--color-font-secondary: #909090;
+	--color-background: #ffffff;
```

Comment:
if you have two backgrounds in the app consider to create `--color-background-primary` and `--color-background-secondary`

`--color-background-primary` — main app background
`--color-background-secondary` — background for some block on the page, e.g. sign in form